apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/transaction_test.exs

```diff
@@ -1357,6 +1267,419 @@ defmodule OMG.WatcherRPC.Web.Controller.TransactionTest do
       |> TestServer.make_response()
       |> TestServer.with_response(context, "/fees.all")
     end
+
+    defp max_amount_spendable_in_single_tx(address, token) do
+      currency = Encoding.to_hex(token)
+
+      address
+      |> WatcherHelper.get_utxos()
+      |> Stream.filter(&(&1["currency"] == currency))
+      |> Enum.sort_by(& &1["amount"], &>=/2)
+      |> Stream.take(Transaction.Payment.max_inputs())
+      |> Stream.map(& &1["amount"])
+      |> Enum.sum()
+    end
+  end
+
+  describe "/transaction.create stealth add inputs" do
+    setup tags do
+      context = TestServer.start()
+      on_exit(fn -> TestServer.stop(context) end)
+      Map.put(tags, :test_server, context)
+    end
+
+    @tag fixtures: [:phoenix_ecto_sandbox]
+    test "does not stealth add inputs where the number of inputs reached maximum", %{
```

Comment:
can you elaborate what this fraze means? `does not stealth add inputs`?

Reply:
we use `stealth` word when we're going to select extra inputs just for merging (optimising) it.